### ISSUE TITLE
allow using ceph secret 

### ISSUE THREAD

**Issue by rootfs** [yp-engineering/rbd-docker-plugin#10]
https://github.com/yp-engineering/rbd-docker-plugin/blob/master/driver.go#L815 assumes keyrings can be found at /etc/ceph/. This is sometimes impractical in a large scale setup where syncing `/etc/ceph` is not possible.

rbd can also take a secret, rather than looking up at `/etc/ceph`. An example can be found at https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/rbd/rbd_util.go#L231

**Comment by hekaldama** [comment]
Good idea. Will look at this and how we can all collaborate instead of making our own rbd talkers. Will continue to push on https://github.com/contiv/volplugin/issues/10.

**Comment by ps-account** [comment]
I think it would also be nice to be able to do this during the "docker run"  invocation. But that could be a separate enhancement.
